src/binder/clrprivbinderassemblyloadcontext.cpp

```diff
@@ -244,7 +271,7 @@ HRESULT CLRPrivBinderAssemblyLoadContext::SetupContext(DWORD      dwAppDomainId,
             if(SUCCEEDED(hr))
             {
                 // Save the reference to the AppDomain in which the binder lives
-                pBinder->m_appContext.SetAppDomainId(dwAppDomainId);
+                pBinder->m_appContext.SetAppDomainId(pAppDomain->GetId().m_dwId);
```

Comment:
Is there a specific reason we need to pass AppDomain* to get the ID, instead of just passing the ID?